ModelBase/test/NotificationsTest.cpp

```diff
@@ -90,15 +90,15 @@ TEST(ModelBasePlugin, ModificationNotificationTests)
 	CHECK_CONDITION(nl.removedNodes.contains( left));
 	CHECK_CONDITION(nl.removedNodes.contains( left->name()));
 
-	TestNodes::PositionExtension* posLeft = left->extension<TestNodes::PositionExtension>();
+	auto posLeft = left->extension<TestNodes::PositionExtension>();
 	CHECK_CONDITION(nl.removedNodes.contains( posLeft->xNode()) );
 	CHECK_CONDITION(nl.removedNodes.contains( posLeft->yNode()) );
 
-	TestNodes::PositionExtension* posRight = right->extension<TestNodes::PositionExtension>();
+	auto posRight = right->extension<TestNodes::PositionExtension>();
 	CHECK_CONDITION(nl.removedNodes.contains( posRight->xNode()) );
 	CHECK_CONDITION(nl.removedNodes.contains( posRight->yNode()) );
 
-	SAFE_DELETE(posLeft);
+	posLeft.reset();
```

Comment:
You can even remove this line, it doesn't seem to be needed now (it was used for deletion before).